### PR TITLE
update ca and eoa versions in constants

### LINKED ISSUE #481: feat: modify CA_VERSION in constants
# Feature Request

Modify CA_VERSION value using pedersen_hash of "KAKAROT_CA_V0.0.1" and replace EOA_VERSION value by pedersen hash of "KAKAROT_EOA_V0.0.1" (instead of "KAKAROT_AA_V0.0.1")

**Related Code**

`constants.cairo`

### PR DESCRIPTION
**Pull request by hurrikaanig**
Time spent on this PR: 0.1 day

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

Resolves #481

### CODE REVIEW

File: src/kakarot/constants.cairo [hunk 1]
```diff
@@ -62,8 +62,10 @@ namespace Constants {
     // PROXY
     const INITIALIZE_SELECTOR = 0x79dc0da7c54b95f10aa182ad0a46400db63156920adb65eca2654c0945a463;
     const CONTRACT_ADDRESS_PREFIX = 'STARKNET_CONTRACT_ADDRESS';
-    const EOA_VERSION = 0x11F2231B9D464344B81D536A50553D6281A9FA0FA37EF9AC2B9E076729AEFAA;  // pedersen("KAKAROT_AA_V0.0.1")
-    const CA_VERSION = 10;
+    // hex(pedersen_hash(69796595864846484649,, 0)) ASCII decomposition of EOA_V0.0.1
```
Comment (ClementWalter) [inline src/kakarot/constants.cairo:65]: this comment is misleading. Actually:

- `EOA_V0.0.1` => `[69, 79, 65, 95, 86, 48, 46, 48, 46, 49]` => `0x454f415f56302e302e31` = `327305290678328760479281`

don't know what you computed actually

File: src/kakarot/constants.cairo [hunk 2]
```diff
@@ -62,8 +62,10 @@ namespace Constants {
...
+    const EOA_VERSION = 0x8507b3f4ea72db226b3651c971f3462f927422eb0e6c13931c6ba0374f0a4b;
+    // hex(pedersen_hash(676595864846484649, 0)) ASCII decomposition of CA_V0.0.1
+    const CA_VERSION = 0x1fcdf3c8a5164d22002cae6d2bd98d6f1d647ef799fb03bd6658404812189f2;
```
Comment (ClementWalter) [inline src/kakarot/constants.cairo:68]: value is here `0x43415f56302e302e31` = `1240642431438726114865`

**Review by ClementWalter — CHANGES_REQUESTED** [review]

File: src/kakarot/constants.cairo (hunk 1 above)
Comment (hurrikaanig) [reply src/kakarot/constants.cairo:65]: I have concatenated `[69, 79, 65, 95, 86, 48, 46, 48, 46, 49]` => `69796595864846484649` and hashed with pedersen hash. But I can make it this way also if you think it's better

Comment (ClementWalter) [reply src/kakarot/constants.cairo:65]: yes plz, while a hex string is an array of bytes, there's no standard way of going from concatenate array of int to bytes

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**